### ISSUE TITLE
Labels displaying letters in 'random' places

### ISSUE THREAD

**Issue by jeffwhitty** [AnalyticalGraphicsInc/cesium#4716]
I have observed this bizarre behaviour intermittently over a long period. I have updated the cesium base twice and running the ~~latest~~ (~~December~~ November '16) release. This was also observed in the August '16 release and once or twice on a version in between but I can't recall which - point being this appears to have been occurring consistently for a long time.

The issue comes when I add a label to my globe. The first two letters render together in completely the wrong location, then I get a few letters, then miss one, get some, miss some. The 'missing' ones are all neatly lined up at the completely wrong location where the first two letters went, the remainder appear to be in the correct positions.

I have thousands of points on the globe and on mouse hover I display a label with its name. When the error occurs it happens to every single point that I tested (and it was lots). I do notice that the 'missing' letters all appear at roughly the same longitude regardless of the position of the intended point. The latitude does vary slightly in the direction of the intended position (ie if the intended position is north of the incorrect location, the incorrect letters are further north than if the intended position is south.)

I cannot replicate this at will, it doesn't happen all the time, and simply refreshing the window gets the desired behaviour back. Unfortunately people signing off on a billion dollar project don't want to hear "just refresh it and it'll come good". No exaggeration, billion dollar project (not all of those dollars are software, of course).

Attached is a partial screen shot. The label should read "WUGANG ATLANTIC" and is rendered with a small pixeloffset to place it slightly right of the triangle which is selected. I have deliberately taken a sample where the intended position is close to the error position for ease of viewing. If I were to mouse over a vessel that is 100 degrees east of this location the 'missing' letters would still appear where the WU is.

** Apparently I can't upload an attachment from this site, I will email it home for the end of the day.
(edit: removed December and corrected to November release, sorry)

**Comment by emackey** [comment]
I've seen this once, but don't know how to reproduce.  Were you using `HeightReference` other than `NONE` on the labels?  Can you share any code that created the labels, or their settings?

**Comment by hpinkos** [comment]
Hello @jeffwhitty, I believe this issue was fixed in the 1.27 release in November.  Are you able to reproduce this with the latest version of Ceisum?  If so, as @emackey said, please share a code sample to reproduce this.  Thanks!

**Comment by jeffwhitty** [comment]
Hey guys, I have a dozen installations in this environment so there's a chance I'm simply incorrect in which version I saw this in, it certainly happened with the August release. I thought it was my dev server (which is up to November release) that I saw this on most recently but I guess there's a chance I'm mistaken.

Literally just refreshing the page caused the problem and then fixed it again, zero code change. My dev system is isolated for just me at the moment and I happen to be in control of the promotion to other environments so I can absolutely guarantee there were no code changes by other people to contaminate the result. This leads me to suspect something within Cesium iself loads out of order occasionally because my code uses Promises to wait until everything is ready before loading my code in order.

I can't paste the actual code from the other environment so if there's a typo in the below it's not affecting my actual code;
```javascript
(code as per the screenSpaceMouseMoveEventHandler samples to detect the mouse hover, inside which we have the below)
var e = viewer.entities.getById(id);
e.label = new Cesium.LabelGraphics({
  text: 'WUGANG ATLANTIC',
  pixelOffset: new Cesium.Cartesian2(15,0),
  horizontalOrigin: Cesium.HorizontalOrigin.LEFT,
  eyeOffset: new Cesium.Cartesian3(0.0, 0.0, -2000.0),
  font: 'bold 18px Calibri',
  fillColor: color, //color is something like Cesium.Colors.BLUE, stored in a variable
  outlineColor: outline, //like color, something like Cesium.Colors.BLACK
  outlineWidth: 1,
  style: Cesium.LabelStyle.FILL_AND_OUTLINE
});
```

**Comment by hpinkos** [comment]
@jeffwhitty I think I finally figured out what the problem was here, and the fix will be included in the Cesium 1.36 release available on August 1.  Please re-open this issue if you're still experiencing the problem after updating.  Thanks!
